include/cudaq/Optimizer/Transforms/Passes.td

```diff
@@ -796,4 +796,14 @@ def UpdateRegisterNames : Pass<"update-register-names"> {
   }];
 }
 
+def WiresToWiresets : Pass<"wires-to-wiresets", "mlir::ModuleOp"> {
```

Comment:
We could get parallelism if we added the WireSetOp into the ModuleOp but then ran this conversion on all the func::FuncOps in the ModuleOp.